recipe/cross_compile_support.sh

```diff
@@ -105,7 +108,7 @@ if [[ "${HOST_PLATFORM}" != "${BUILD_PLATFORM}" ]]; then
                 sed 's/"//g' versions.txt | sed 's/_/-/g' | sed 's/sanitizer-api/sanitizer/g' | sed 's/-dev/-devel/g' | sed 's/-develel/-devel/g' > rpms.txt
 
                 # filter packages from manifest down to what we need for cross-compilation
-                grep -E "cuda-(compat|cudart|cupti|driver|nvcc|nvml|nvprof|nvrtc|nvtx|profiler).*|lib(cu|npp|nvjpeg).*" rpms.txt > rpms_cc.txt
+                grep -E "cuda-(cccl|compat|cudart|cupti|driver|nvcc|nvml|nvprof|nvrtc|nvtx|profiler).*|lib(cu|npp|nvjpeg).*" rpms.txt > rpms_cc.txt
```

Comment:
Is this what we need?

Reply:
I believe so!

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=802202&view=logs&j=ab84e6e6-af01-55f5-f9b2-33f6b533d940&t=5142a81e-84f2-575a-df84-316804fac885&l=438